### PR TITLE
use images built by cdp all the way

### PR DESCRIPTION
**Pull request by linki**
use images built by cdp all the way

### CODE REVIEW

**Comment by linki** [comment]
These versions are redicilous. I'm gonna tag them and update. Let me know if you disagree.

**Comment by linki** [comment]
Left:
* [x] https://github.bus.zalan.do/teapot/rescheduler/pull/7/files
* [x] ~~https://github.com/hjacobs/kube-aws-autoscaler/pull/32~~
* [x] https://github.bus.zalan.do/teapot/kube-dns-metrics-bash/pull/4

**Comment by Raffo** [comment]
PR for rescheduler is merged.

**Comment by linki** [comment]
Best I can do for now.

File: cluster/manifests/kube-dns-metrics/deployment.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
         version: v1.0.0
     spec:
       containers:
-        - image: pierone.stups.zalan.do/teapot/kube-dns-metrics:v1.0.0
+        - image: pierone.stups.zalan.do/teapot/kube-dns-metrics:v1.0.0-11-g406577e
```
Comment (Raffo) [inline cluster/manifests/kube-dns-metrics/deployment.yaml]: Did you miss this or is it on purpose?

Comment (linki) [reply cluster/manifests/kube-dns-metrics/deployment.yaml]: missed it

**Comment by Raffo** [comment]
I think we are good to add the ready to test label.

**Comment by linki** [comment]
updated image policy webhook to allow images from `gcr.io/google-containers`. let's see.

**Comment by szuecs** [comment]
:+1:

**Comment by linki** [comment]
👍

**Comment by linki** [comment]
sweet 🎉

**Pull request merged**